helm/stargate/templates/stargate_coordinator_deployment.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: stargate-coordinator
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: coordinator
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: coordinator
+  template:
+    metadata:
+      labels:
+        app: coordinator
+    spec:
+      containers:
+      - name: coordinator
+        {{ if eq .Values.cassandra.clusterVersion "3.11" }}
+        image: "{{ .Values.image.repository311 }}:{{ .Values.image.tag }}" 
+        {{ else if eq .Values.cassandra.clusterVersion "6.8" }}
+        image: "{{ .Values.image.repositoryDse68 }}:{{ .Values.image.tag }}"
+        {{ else }}
+        image: "{{ .Values.image.repository40 }}:{{ .Values.image.tag }}"
+        {{end}}
+        ports:
+        - containerPort: 8091
+        - containerPort: 8090
+        - containerPort: 9042
+        - containerPort: 8081
+        args:          
+         {{  range .Values.arguments }}
+           - {{ . }}
+         {{ end }}
+        readinessProbe:
```

Comment:
I would consider making probes configurable. As I mentioned below with resources, I would simply allow for a yaml block.

Reply:
Done